main.go

```diff
@@ -110,5 +115,23 @@ func main() {
 		log.Fatal(err)
 	}
 
+	if opts.ProfilerOutput != "" {
```

Comment:
I still think we should not expose this functionality this way. Let's remove it and figure out why net/http/pprof does not work for you

Reply:
Done